components/brave_wallet/browser/brave_wallet_utils.cc

```diff
@@ -929,29 +891,19 @@ mojom::NetworkInfoPtr GetChain(PrefService* prefs,
   return nullptr;
 }
 
-GURL GetInfuraURLForKnownChainId(const std::string& chain_id) {
-  auto endpoint = brave_wallet::GetInfuraEndpointForKnownChainId(chain_id);
-  if (!endpoint.empty()) {
-    return GURL(endpoint);
-  }
-
-  auto subdomain = brave_wallet::GetInfuraSubdomainForKnownChainId(chain_id);
-  if (subdomain.empty()) {
-    return GURL();
-  }
-  return GURL(
-      base::StringPrintf("https://%s-infura.brave.com/", subdomain.c_str()));
+bool IsEndpointUsingBraveProxy(const GURL& url) {
```

Comment:
Let's name it IsEndpointUsingBraveWalletProxy

Reply:
https://github.com/brave/brave-core/pull/24045/commits/36124d274325debcadb2a917362ac02fde9a4c0d